app/utils/avoid_native_bridge.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Needs the license header. I need to see if Jesse is going to be adding the ESLint plugin over here that forces a header on all files